### ISSUE TITLE
tick box to select >3 moves left

### ISSUE THREAD

**Issue by Loreinator** [Loreinator/Shuffle-Move#7]
[AvatarMew:
Will there be an tick box to select >3 moves left for abilities like last ditch effort?](https://www.reddit.com/r/ShuffleMove/comments/3h1wpt/release_shuffle_move_v0316_is_released/cu3x5qh)

**Comment by Loreinator** [comment]
This will probably get in as a simple checkbox in the move chooser somewhere, Most likely the best spot will end up being an option in the menu bar. This also allows simple code reuse and it can be included in the move menu for the main window as well. The value could be stored in the preferences manager (global settings) and reset when a new stage is selected, which should provide a moderately intuitive usage for the option itself.

Hopefully there aren't any real issues for implementing the abilities, [although the data here](http://pastebin.com/TEGuv3Ud) should make it quick to write them in. There will need to be a new pathway through from the general preferences to the simulation core to check for the activation of this option, but that should be simple enough.

**Comment by Loreinator** [comment]
Currently working on this by implementing stage move and health tracking, to better address the underlying issue. Thankfully I have this weekend free to do most of the real work, so it probably will be done soon.

**Comment by Loreinator** [comment]
Fixed in the development branch, ready for v0.3.17.

The fix includes data for the main stages plus victini and meowth for both moves left and health left. This is used to model the abilities which depend on those statistics and so those abilities are now included as well.

As I was working on it, it seemed like the very best place to put such data is inside the preferences manager, controlled by stuff at the bottom of the paint pallet (under the current options). I also managed to fix a scroll bug with the paint pallet itself while I was in there.
